src/qt/bitcoin.cpp

```diff
@@ -46,6 +46,7 @@
 #include <QThread>
 #include <QTimer>
 #include <QTranslator>
+#include <QString>
```

Comment:
nit: Could you apply [clang-format-diff.py](https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy) to your commit?